docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -10,12 +10,12 @@ between two different SQL Server instances.
 Requirements
 ------------
 
-Requirements for using the connector in a catalog to connect to a SQL Server
-data source are:
+To connect to a SQL Server data source using the connector in a catalog, you
+need:
 
-* SQL Server 2012 or higher, or Azure SQL Database
-* Network access, by default on port 1433, from the Trino coordinator and
-  workers to SQL Server.
+* SQL Server 2012 or higher, or Azure SQL Database.
```

Comment:
Azure SQL Database is just hosted SQL Server IIRC. In same vein we could mention Google Cloud SQL or Amazon RDS too.

We should document this in some other way IMO. Maybe some text like `Hosted solutions should also work as long as the above requirements are met.`?

Reply:
we're putting this in future work, thanks.